components/CodeSyntax.jsx

```diff
@@ -44,7 +44,7 @@ const copyToClipboard = str => {
 }
 
 
-const CodeSyntax = ({ children, highlightedLines, language, showLineNumbers }) => {
+const CodeSyntax = ({ innerID, children, highlightedLines, language, showLineNumbers }) => {
 
```

Comment:
why not just id?

Reply:
It sounded like @rhyolight wanted it named in a way that was suggestive of it being an id for the inner div. It would also prevent any future react update from checking for the `id` prop and setting it to the parent leading to duplicated ids (not sure how valid this is, but it was a thought I had while naming it). I can change it over to `id` if you guys want, just let me know.